auth/auth-router.js

```diff
@@ -3,7 +3,7 @@ const bcrypt = require("bcryptjs");
 const middleware = require("./middleware.js");
 const createToken = require("./createToken.js");
 
-const Users = require("../users/writersModels");
+const Users = require("../users/writer_profiles/writersProfileModel");
```

Comment:
Maybe if we were to separate the specific user onboarding code into its own file (like a general userModel file) we would be able to use this code for onboarding of both use types?